docs/book/src/developer/providers/v1.2-to-v1.3.md

```diff
@@ -12,7 +12,7 @@ maintainers of providers and consumers of our Go API.
 **Note**: Only the most relevant dependencies are listed, `k8s.io/` and `ginkgo`/`gomega` dependencies
 in Cluster API are kept in sync with the versions used by `sigs.k8s.io/controller-runtime`.
 
--
+- sigs.k8s.io/kind: v0.14.x => v0.15.x
```

Comment:
Q: is there some breaking change in kind forcing up to use the latest version or it is just a "regular" version bump?

Reply:
It is because this version adds support for Kubernetes 1.25

https://github.com/kubernetes-sigs/kind/releases/tag/v0.15.0

Reply:
Thanks, it seems a "regular" version bump

Reply:
Yup + potentially some incompatibilities with old images. Not sure how to interpet the release notes exactly